core/src/test/scala/org/apache/spark/deploy/rest/SubmitRestProtocolSuite.scala

```diff
@@ -86,6 +86,8 @@ class SubmitRestProtocolSuite extends SparkFunSuite {
     message.clientSparkVersion = "1.2.3"
     message.appResource = "honey-walnut-cherry.jar"
     message.mainClass = "org.apache.spark.examples.SparkPie"
+    message.appArgs = Array("hdfs://tmp/auth")
+    message.environmentVariables = Map("SPARK_SCALA_VERSION" -> "2.11")
```

Comment:
we might be able to set this here - we need to have it working with multiple versions of scala

Reply:
In this case it's just a name, I could change it in order to make it clear!